### PR TITLE
PTX-1304 fix a torpedo bug which makes internal kvdb cluster go out of quorum causing cores

### PR DESCRIPTION
**Pull request by thiguetta**

### CODE REVIEW

**Comment by harsh-px** [comment]
The PR still doesn't fix the internal kvdb issue.

It just fixes the torpedo logic to select nodes that can be down. Internal kvdb will require another fix. Please rename the PR title and commit messages to reduce confusion.

File: drivers/volume/portworx/portworx.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ const (
 	maintenanceWaitTimeout           = 2 * time.Minute
 	inspectVolumeTimeout             = 10 * time.Second
 	inspectVolumeRetryInterval       = 2 * time.Second
-	validateDeleteVolumeTimeout      = 3 * time.Minute
+	validateDeleteVolumeTimeout      = 5 * time.Minute
```
Comment (harsh-px) [inline drivers/volume/portworx/portworx.go]: why is the timeout bumped? A PVC should get deleted in 3 minutes.

Comment (thiguetta) [reply drivers/volume/portworx/portworx.go]: sometime it's taking longer than that, actually no error for that

Comment (thiguetta) [reply drivers/volume/portworx/portworx.go]: actually this piece of code belong to another ticket I only put in here to test three problems at once

Comment (harsh-px) [reply drivers/volume/portworx/portworx.go]: revert this timeout. If deletes are timing out, we need to root cause the actual issue in PX.

Comment (thiguetta) [reply drivers/volume/portworx/portworx.go]: ok, I might need some help on that since I haven't found any error in the logs

Comment (harsh-px) [reply drivers/volume/portworx/portworx.go]: Sure, I can help take a look if you have a cluster where PVC delete took more than 3 minutes. First place to look is the controller manager logs which will have the errors reported on the PVC delete call to Portworx

File: tests/basic/basic_test.go [hunk 2]
```diff
@@ -209,19 +209,20 @@ func getNodesThatCanBeDown(ctx *scheduler.Context) []node.Node {
 			Expect(err).NotTo(HaveOccurred())
 			Expect(replicas).NotTo(BeEmpty())
 			// at least n-1 nodes with replica need to be up
-			for i := 0; i <= len(replicas)-1; i++ {
-				nodesThatCantBeDown[replicas[i]] = true
+			maxNodesToBeDown := len(replicas)/2 + 1
```
Comment (harsh-px) [inline tests/basic/basic_test.go]: for a repl 2 volume, this logic will shutdown both replica nodes.

**Pull request merged**